### DISCUSSION TITLE
Suricata 7.0.8 released

### DISCUSSION THREAD

**Discussion by Lesterol** [Security-Onion-Solutions/securityonion#14030] [2.4] [answered]
### Version

2.4.110

### Installation Method

Security Onion ISO image

### Description

upgrading

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

-

### RAM

-

### Storage for /

-

### Storage for /nsm

-

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hi guys!
I heard there was an update for Suricata that fixes serious vulnerabilities. Maybe add this update in the next SO update?
https://forum.suricata.io/t/suricata-7-0-8-released/5137

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by zfcarsonb** [comment, marked as answer]
I saw an issue (#14024) referencing this earlier this week. I assume a patch release will come out sometime soon.

**Reply by dougburks** [reply]
Yes, we are working on version 2.4.111 right now.

**Reply by dougburks** [reply]
Security Onion 2.4.111 is now available and includes Suricata 7.0.8:
https://blog.securityonion.net/2024/12/security-onion-24111-now-available.html
